gcp/check_production_runs_status

```diff
@@ -90,19 +91,19 @@ main () {
     while read -r run_info; do
         sample_name=$(echo "${run_info}" | cut -f1)
         set_name=$(echo "${run_info}" | cut -f2)
-        bucket=$(echo "${run_info}" | cut -f3)
+        entity=$(echo "${run_info}" | cut -f3)
         exists_in_success_run_dirs=$(find "${find_dirs[@]}" -maxdepth 1 -type d -name "${set_name}" | wc -l);
         if [[ ${exists_in_success_run_dirs} == 0 ]]; then
             if [[ ${set_name} =~ "_HMFregVAL_" ]]; then
                 echo "# This is a validation run, so can be ignored for production ${set_name}"
-            elif [[ "${bucket}" == "hmf-output-services" && ! "${sample_name}" =~ ^CORE.{8}T ]]; then
+            elif [[ printf '%s\n' "${non_db_entities[@]}" | grep -P "^${entity}$" && ! "${sample_name}" =~ ^CORE.{8}T ]]; then
```

Comment:
Maybe cleaner to use a query on entity itself instead of printf+grep? Like `add_to_database=$(hmf_api_get "entities?name=${entity}" | jq -r '.[-1].add_to_db')`

Reply:
Yeah I wish there was a clean way to look for array membership in `bash`. Just thinking through the way you're suggesting, I guess I'd set the variable up above and then do like a `$add_to_database eq "0"` in the conditional?